astrocloud cli 1.5.0 (#639)
astronomer/docs: astro/cli-reference/astrocloud-deployment-variable-update.md

---
sidebar_label: "astrocloud deployment variable update"
title: "astrocloud deployment variable update"
id: astrocloud-deployment-variable-update
description: Reference documentation for astrocloud deployment variable update.
---

## Description

> For a given Deployment on Astro, use `astrocloud deployment variable update` to update the value of an existing environment variable via the Astro CLI. You can do so either by:
> 
> - Manually entering a new `--value` for a given `--key` in your command line
> - Modifying the value of any environment variable in a `.env` file and loading that change with `--load`

Reviewer: ```suggestion
For a given Deployment on Astro, use `astrocloud deployment variable update` to update the value of an existing environment variable via the Astro CLI. To do so, you can either:

- Manually enter a new `--value` in your command line for a given `--key`, or
- Modify the value of one or more existing environment variables in a `.env` file and load that file with `--load`.
```
